city_scrapers/spiders/det_employment_solutions.py

```diff
@@ -0,0 +1,184 @@
+                return COMMITTEE
+
+        return BOARD
+
+    def find_date(self, item, meeting_type, is_start=False):
+        """Parses item and creates a datetime formatted string."""
+        if meeting_type is None:
+            return None
+        if meeting_type == "MWDB":
+            # remove the 'nd' 'rd' 'th' from number day
+            caldate = item.css("strong ::text").get().strip()[:-2]
+            timeframe = item.css("p ::text").get().strip().split(" - ")
+            # webpage dates don't include year, assume year of now
+            curr_year = str(datetime.now().year)
+            curr_month = str(datetime.now().month)
+            # If the current month is not January but January is in the meeting
+            # date than the meeting date falls in the next year.
+            if curr_month != "1" and "January" in caldate:
+                curr_year = str(int(curr_year) + 1)
+            meeting_time = timeframe[0] if is_start else timeframe[1]
+            date_str = "{} {} {}".format(caldate, curr_year, meeting_time)
+            date_str_fmt = "%A, %B %d %Y %I:%M%p"
+        if meeting_type == "CEAC":
+            # date is in the format: 11/23/2020 – 11 am – 12:30 pm
+            parsed_item = re.compile("(: )|( – )").split(
+                item.css("ul > li ::text").get().strip()
+            )
+            caldate = parsed_item[0].split("/")
+            timeframe = re.compile("(-)|(–)").split(parsed_item[3])
+            if len(timeframe) < 2 and (not is_start):
+                return None
+            meeting_time = timeframe[0] if is_start else timeframe[3]
+            if ":" not in meeting_time:
+                meeting_time = ":00 ".join(meeting_time.split(" "))
+            if len(caldate[2]) == 2:
+                caldate[2] = "20" + caldate[2]
+            date_str = "{} {}".format("/".join(caldate), meeting_time.replace(" ", ""))
+            date_str_fmt = "%m/%d/%Y %I:%M%p"
+        if meeting_type == "DESC":
+            if not is_start:
+                return None
+            date_str = "9:45 am"
+            date_str_fmt = "%I:%M %p"
+
+        return datetime.strptime(date_str, date_str_fmt)
+
+    def _parse_start(self, item, meeting_type):
+        """Parse start datetime as a naive datetime object."""
+        return self.find_date(item, meeting_type, True)
+
+    def _parse_end(self, item, meeting_type):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return self.find_date(item, meeting_type)
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return "Please check pdf schedule for exact time of DESC meetings."
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        location_info = item.css("p ::text").get()
+        first_comma_index = location_info.find(", ")
+        return {
+            "address": location_info[first_comma_index + 3 :],
```

Comment:
If possible it would be good to remove the "(location subject to change)" here

Reply:
removed the "(location subject to change)" line in 25d0aef3b3cb67b4bb2b81b9939ae0a691fb42d9